src/components/modals/AddressModal.tsx

```diff
@@ -144,7 +145,7 @@ export class AddressModalComponent extends React.Component<Props, State> {
 
   checkIfEnsDomain = (name: string): boolean => ENS_DOMAINS.some(domain => name.endsWith(domain))
 
-  fetchDomain = async (domain: string, currencyTicker: string): Promise<string> => {
+  fetchUnstoppableDomainAddress = async (domain: string, currencyTicker: string): Promise<string> => {
```

Comment:
Actually one other thing. Since these are separate functions now you can remove the ENS functionality from this function and leave it as UD only. Lines 150 through 154.